extendr-api/src/wrapper/environment.rs

```diff
@@ -6,6 +6,45 @@ pub struct Environment {
 }
 
 impl Environment {
+    /// Create a new, empty environment parented on global_env()
+    /// ```
+    /// use extendr_api::prelude::*;
+    /// test! {
+    ///     let env = Environment::new();
+    ///     assert_eq!(env.len(), 0);
+    /// }
+    /// ```
+    pub fn new() -> Self {
+        // 14 is a reasonable default.
+        Environment::new_with_capacity(14)
+    }
```

Comment:
Is there any reason we don't have a `new()` function that allows us to specify a parent environment? That's a very common use case.

Reply:
I'll put one in...

Reply:
Note that the parent must be an Environment. This should tighten things up and reduce checks.